newIDE/app/src/SceneEditor/index.js

```diff
@@ -1107,36 +1116,39 @@ export default class SceneEditor extends React.Component<Props, State> {
         {
           label: i18n._(
             t`Edit object ${shortenString(
-              this.state.selectedObjectNames[0],
+              this.state.selectedObjects[0].object.getName(),
```

Comment:
I think you can compute the object name at the start of the method and then use it in the items that need it